### ISSUE TITLE
Install with multiple disks promises to permit review of selection but proceeds immediately to creating partitions on the disk

### ISSUE THREAD

**Issue by stephendee** [openmediavault/openmediavault#205]
During my installation of OMV 4.1.3 on an Asus P5Q-Pro (E8500) using an IDE DVD, the install program alerted me to the fact that there were two disk drives installed and that I would be given an opportunity to verify my selection after selecting the drive. That opportunity was not provided and the install proceeded to wipe the disk. In my case, I had the right disk, so I didn't suffer any loss, but this is the sort of thing that could be catastrophic for someone who read and understood the screen text. In other installations, I've seen the install program provide the installer with the option of proceeding to a partition manipulation program for custom partitioning, at which point, it would be pretty easy for someone to notice they were probably looking at the wrong drive.

**Comment by esbeeb** [comment]
Yes, I can verify that it is just as @stephendee says.  And I did blow up a couple of partitions I didn't want to lose, resulting in lost data!!  Aargh!!  Please add a final safety check before proceeding with the disk format, warning that all partitions on that disk will be lost.  Please be true to the message that says "...but not before you have confirmed that you really want to make the changes".  On the screenshot attached, once you punch enter, any and all partitions on the selected disk are blasted without further warning!
![DSCF8317](https://user-images.githubusercontent.com/33113557/57063570-482a5800-6cf6-11e9-90a4-9c5ae0b8dc5c.JPG)

**Comment by votdev** [comment]
You ignored the warning message that you should disconnect all devices except the one for the root file system.

**Comment by votdev** [comment]
It is also clearly mentioned to disconnect the data devices in the [docs](https://openmediavault.readthedocs.io/en/latest/installation/index.html).

**Comment by esbeeb** [comment]
It wasn't that I accidentally blew up the wrong disk, it was that I didn't get the message that "all data on the disk you select" meant "OMV doesn't just install to one partition of your choosing, it takes over a whole disk to itself, and there can't be other partitions on that disk.". Virtually all other Linux distros I've ever installed before (albeit they were not NAS-specific distros) always let me just install to one partition, respecting other existing partitions.  So that expectation of "my other partitions on that disk ought to remain safe, as surely I'll get a chance to choose just one partition" was the problem, which I feel is not unreasonable since the message "but not before you have confirmed that you want to make the changes" gave me a false sense of security.

Maybe your wording of "Select disk to partition" could be restated in a less ambiguous way, saying something like "Select disk to completely wipe clean and entirely re-partition."

**Comment by votdev** [comment]
Please note that OMV uses Debian installer, so the wording in dialogs can not be changed.

**Comment by votdev** [comment]
> Virtually all other Linux distros I've ever installed before (albeit they were not NAS-specific distros) always let me just install to one partition, respecting other existing partitions.

OMV does not allow to store user data on the system device by design. Because of that it does not make sense to allow the user to choose a partition during the installation process.

**Comment by votdev** [comment]
The [docs](https://openmediavault.readthedocs.io/en/latest/installation/via_iso.html) have been adapted.
